### PR TITLE
Simple orderly_run

### PR DESCRIPTION
**Pull request by richfitz**
Closer to the orderly MVP - a simple copy of orderly_run. Does not handle dependencies, partly because that requires the query interface for outpack to exist (both the language and some way of pulling this information out nicely). The docs and comments link to other not-yet-implemented bits and differences with orderly

I've managed to preserve the draft/archive distinction except that a successful run *always* moves to the archive. This is going to lead to a mixed bag in the archive directory (stuff that is shared with the server, stuff that is local) but that will end up reflected in outpack's metadata. If that is not going to work then outpack will need multiple archive directories, which will be a bit weird but doable.

### CODE REVIEW

File: R/orderly.R [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+    ##
+    ## TODO: once mrc-3382 is fixed we can mark these to ignore in
+    ## outpack
+    message("Some extra files found: ",
+            paste(squote(extra), collapse = ", "))
+  }
+}
+
+
+check_parameters <- function(parameters, info) {
```
Comment (hillalex) [inline R/orderly.R:164]: I'm a bit confused by the splitting of responsibilities between `outpack` and `orderly2` - why can the former not do this kind of parameter validation?

Comment (richfitz) [reply R/orderly.R:164]: oh good point, and it turns out that it *does* do that validation. Looks like I've copied too much over from orderly, will trim that out.

Comment (richfitz) [reply R/orderly.R:164]: ah, there's also stuff that's orderly specific: orderly(2) has a fixed idea of what parameters are expected (in orderly.yml) but outpack does not care - it just wants a list. So we need to keep the validation against the parameter spec here, but drop the bits where we check that the given set are sensible, which is much less code

**Review by r-ash — APPROVED** [review]
This looks good. It leaves a copy of the report artefacts in `draft` dir as well as copying them to `archive` is that intended? Do we want to remove the `draft` files once the run has completed successfully?

**Review by hillalex — APPROVED** [review]

**Comment by richfitz** [comment]
> This looks good. It leaves a copy of the report artefacts in draft dir as well as copying them to archive is that intended? Do we want to remove the draft files once the run has completed successfully?

You're right - this is something to think about as we probably should delete that. It interacts a bit with one of the other bits I need to pull together so held off on sorting it right now (VIMC-6645, mrc-3382)

**Pull request merged**